spec/message.abnf

```diff
@@ -1,11 +1,11 @@
 message           = simple-message / complex-message
 
-simple-message    = [simple-start pattern]
+simple-message    = [s] [simple-start pattern]
```

Comment:
I don't think this works because:

```abnf
simple-start-char = content-char / s / "@" / "|"
```

This contains `s`, so allowing `simple-start-char` to match the optional spaces we just installed--and allowing access to `text-char` in `pattern`.

I think we might simplify by changing simple-start-char:

```abnf
simple-start-char = [s] (content-char / "@" / "|")
```

Reply:
Ah, you're right. Should be fixed now with `s` removed from `simple-start-char`.